test-suite/testkit/src/builder.rs

```diff
@@ -265,14 +268,28 @@ impl TestKitBuilder {
             })
             .build();
 
-        TestKit::assemble(
-            TemporaryDB::new(),
-            network,
-            genesis_config,
-            self.additional_runtimes,
-            self.plugins,
-            self.api_notifier_channel,
-        )
+        #[cfg(feature = "exonum-node")]
```

Comment:
Nit: I'd prefer to have `cfg` on a function instead of code blocks, something like:

```rust
#[cfg(feature = "exonum-node")]
fn complete_initalization(self, teskit: mut Testkit) -> Teskit {
    testkit.set_plugins(self.plugins);
    testkit
}

#[cfg(not(feature = "exonum-node"))]
fn complete_initalization(self, teskit: Testkit) -> Teskit {
    testkit
}
```

and here just:


```rust
let testkit = TestKit::assemble(
    TemporaryDB::new(),
    network,
    genesis_config,
    self.additional_runtimes,
    self.api_notifier_channel,
);
self.complete_initalization(testkit)
```

Reply:
`self` is partially moved by this point, and the `plugins` field only exists if the `exonum-node` feature is on. Thus, conditionals on the call site seem inevitable.

Reply:
Sad.